assets/demos/DemoApp.Tests.ps1

```diff
@@ -1,107 +1,132 @@
 BeforeAll {
-  Import-Module "../../BenchPress/Helpers/BenchPress.Azure/BenchPress.Azure.psd1"
+  Import-Module "../../bin/BenchPress.Azure.psd1"
```

Comment:
Can you update the README in this folder to point to the `installation.md` so that it's clear that users will need to build the PowerShell module first in order for this import to work?

Reply:
Done!